test/functional/rpc_rawtransaction.py

```diff
@@ -178,6 +182,67 @@ def getrawtransaction_tests(self):
         block = self.nodes[0].getblock(self.nodes[0].getblockhash(0))
         assert_raises_rpc_error(-5, "The genesis block coinbase is not considered an ordinary transaction", self.nodes[0].getrawtransaction, block['merkleroot'])
 
+    def getrawtransaction_verbosity_tests(self):
+        tx = self.nodes[2].sendtoaddress(self.nodes[1].getnewaddress(), 1)
+        [block1] = self.generate(self.nodes[2], 1)
+        fields = [
+            'blockhash',
+            'blocktime',
+            'confirmations',
+            'hash',
+            'hex',
+            'in_active_chain',
+            'locktime',
+            'size',
+            'time',
+            'txid',
+            'vin',
+            'vout',
+            'vsize',
+            'weight'
```

Comment:
nit: In Python, it's recommended to add trailing commas on multiline lists to minimize the git diff when new items are added later on.
```suggestion
            'weight',
```